service/implementation/auto_request_api/logic_auto_request.py

```diff
@@ -3,11 +3,23 @@
 import requests
 from datetime import datetime
 from typing import Dict
+from sqlalchemy import text
+from database.session import SessionLocal
 from database.azure_blob_storage.save_get_blob import blob_autosave_api
 
 load_dotenv()
-api_key = [os.getenv("APIKEY"), "API_KEY_2", "API_KEY_3"]
-current_key_index = 0
+api_key = [
```

Comment:
Holy mother of Jesus

Reply:
We a poor student, nothing to do with it xD